app/src/main/java/com/example/gdsc/FragmentActivity.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.gdsc
+
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+
+class FragmentActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_fragment)
+
+        // BottomNavigationView 객체 생성
+        var bnv = findViewById(R.id.navigationView) as BottomNavigationView
```

Comment:
```suggestion
        val bnv = findViewById<BottomNavigationView>(R.id.navigationView)
```
네비게이션 객체가 바뀔 일은 거의 없을 것 같아서 val로 변경해도 될 것 같아요!